examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -1869,7 +1868,7 @@ impl FrontendNode {
             .map_err(from_proto)?;
         let mut count = 0;
         loop {
-            match receiver.receive() {
+            match oak::io::receive(&receiver) {
```

Comment:
I think for the SDK crate we can actually recover most of the existing semantics by declaring an extension that exposes the `.receive` and other methods directly on the sender / receiver types (the usual pattern is to have a `ReceiverExt` and `SenderExt` trait, but the naming does not really matter).

Reply:
I changed it to use extension traits as you suggested.